### ISSUE TITLE
Refresh github action is failing

### ISSUE THREAD

**Issue by GenevieveBuckley** [dask/dask-blog#123]
**What happened**:
The "Refresh" github action has been failing for the last 8 months. This GitHub Actions workflow is supposed to cause the GitHub Pages site for the Dask blog to be rebuilt every day at 3am UTC.

https://github.com/dask/dask-blog/blob/gh-pages/.github/workflows/refresh.yml

**What you expected to happen**:
I expected the github action to rebuild/refresh the dask blog website instead of failing.

**Minimal Complete Verifiable Example**:
Here are the logs for the latest failed action: https://github.com/dask/dask-blog/runs/4403644252?check_suite_focus=true

It fails on the step "Trigger GitHub pages rebuild"
```
Run curl --fail --request POST \
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 401 
Error: Process completed with exit code 22.
```

**Anything else we need to know?**:
The last successful refresh action [ran 8 months ago](https://github.com/dask/dask-blog/actions/runs/754210347). I'm guessing something about the github actions environment changed around that time.

cc @jacobtomlinson - maybe you have some suggestions? As the author of https://github.com/dask/dask-blog/pull/69 you have some good background context on this.

**Environment**:
Using Github actions associated with this repository.

<details>
<summary>Click to expand - "Set up job" information from the github action:</summary>

```
Current runner version: '2.285.0'
Operating System
  Ubuntu
  20.04.3
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20211129.1
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20211129.1/images/linux/Ubuntu2004-README.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20211129.1
Virtual Environment Provisioner
  1.0.0.0-master-20211123-1
GITHUB_TOKEN Permissions
  Actions: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
Secret source: Actions
Prepare workflow directory
Prepare all required actions
```

</details>

**Comment by GenevieveBuckley** [comment]
The [401 url error](https://www.lifewire.com/401-unauthorized-error-what-it-is-and-how-to-fix-it-2622934) seems to suggest that something has gone wrong with the github authentication tokens in the github action script.

I have write permissions, but not admin permissions to this repository, so I can't see the github tokens in the repo settings. Perhaps @jacobtomlinson or @jsignell could take a look?

**Comment by jakirkham** [comment]
Just for context [HTTP 401 means Unauthorized]( https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401 ). Likely whatever we are doing needs a GH Token (assuming it is still a permitted action)
